backend/src/modules/ai/test.py

```
@@ -0,0 +1,22 @@
+        self.name = "TestAI"
+        controller.subscribe(
+            modulename="ai",
+            config_events={"ai": self.config_update},
+            message_events={"chat": self.handle_chat_message}
+        )
+
+    def config_update(self, new_config: Any) -> None:
+        if new_config == "1":
+            log.info(f"{self.name} received new config: {new_config}")
+        else:
+            raise ValueError("Invalid config value for TestAI")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)